data/meson.build

```
@@ -0,0 +1,47 @@
+subdir('ui')
+
+desktop_file = i18n.merge_file(
+    input: 'pcmanfm.desktop.in',
+    output: 'pcmanfm.desktop',
+    type: 'desktop',
+    po_dir: '../po',
```

I prefer having it relative to the source root (also below):

```suggestion
    po_dir: meson.source_root() / 'po'
```